src/cloud-api-adaptor/pkg/securecomms/kubemgr/kubemgr.go

```diff
@@ -19,6 +18,8 @@ import (
 	"k8s.io/client-go/util/homedir"
 )
 
+var logger = sshutil.Logger
```

Comment:
nit, can be squashed into the first commit

Reply:
I cant think of a simple way to squash part of commit x to commit y.
When we finish the review, if it is preferred, we can squash all secure comms commits to  one.

Reply:
`git commit add -p` is your friend here.

Reply:
solved